packages/cli/src/index.ts

```diff
@@ -12,7 +12,7 @@ import init from './commands/init/initCompat';
 import assertRequiredOptions from './tools/assertRequiredOptions';
 import loadConfig from './tools/config';
 
-import pkgJson from '../package.json';
+const pkgJson = require('../package.json');
```

Comment:
TS is able to resolve package.json, but with a require. This is the actual fix, rest is errors that accumulated since the bug started happening